syntaxes/clarity.yaml

```diff
@@ -1,184 +1,186 @@
         name: punctuation.section.expression.begin.clarity
     end: \)(\n)?
     endCaptures:
-      '1':
+      "1":
         name: punctuation.section.expression.end.clarity
-      '2':
+      "2":
         name: meta.after-expression.clarity
     name: meta.expression.clarity
     patterns:
-    - include: '#comment'
-    - include: '#define-map'
-    - begin: >
-        (?x)
-        (?<=\() # preceded by (
-          (define-public|define-private|define-constant|define-read-only)\s+
-          \(    # open parens
-            ([[:alnum:]][[:alnum:]!$%&*+-./:<=>?@^_~]*)
-            ((\s+
-              \( # open parens for parameter/type pair
-              \s*
+      - include: "#comment"
+      - include: "#define-map"
+      - begin: >
+          (?x)
+          (?<=\() # preceded by (
+            (define-public|define-private|define-constant|define-read-only)\s+
+            \(    # open parens
               ([[:alnum:]][[:alnum:]!$%&*+-./:<=>?@^_~]*)
-              \s+
-              ([[:alnum:]][[:alnum:]!$%&*+-./:<=>?@^_~]*)
-              \s*
-              \) # close parens for parameter/type pair
+              ((\s+
+                \( # open parens for parameter/type pair
+                \s*
+                ([[:alnum:]][[:alnum:]!$%&*+-./:<=>?@^_~]*)
+                \s+
+                ([[:alnum:]][[:alnum:]!$%&*+-./:<=>?@^_~]*)
+                \s*
+                \) # close parens for parameter/type pair
+                \s*
+                )*)
               \s*
-              )*)
-            \s*
-          \)    # close parens
-      captures:
-        '1':
-          name: keyword.control.clarity
-        '2':
-          name: entity.name.function.clarity
-        '4':
-          name: variable.parameter.function.clarity
-      end: (?=\))
-      name: meta.declaration.procedure.clarity
-      patterns:
-      - include: '#comment'
-      - include: '#sexp'
-      - include: '#illegal'
-    - begin: (?<=\()(define)\s([[:alnum:]][[:alnum:]!$%&*+-./:<=>?@^_~]*)\s*.*?
-      captures:
-        '1':
-          name: keyword.control.clarity
-        '2':
-          name: variable.other.clarity
-      end: (?=\))
-      name: meta.declaration.variable.clarity
-      patterns:
-      - include: '#constants'
-      - include: '#comment'
-      - include: '#sexp'
-      - include: '#illegal'
-    - include: '#language-functions'
-    - include: '#string'
-    - include: '#constants'
-    - match: (?<=[\(\s])(#\\)(space|newline|tab)(?=[\s\)])
-      name: constant.character.named.clarity
-    - match: (?<=[\(\s])(#\\)x[0-9A-F]{2,4}(?=[\s\)])
-      name: constant.character.hex-literal.clarity
-    - match: (?<=[\(\s])(#\\).(?=[\s\)])
-      name: constant.character.escape.clarity
-    - include: '#sexp'
-    - include: '#illegal'
+            \)    # close parens
+        captures:
+          "1":
+            name: keyword.control.clarity
+          "2":
+            name: entity.name.function.clarity
+          "3":
```

Comment:
this seems to fix an issue where only the last function arg would get highlighted